### PR TITLE
Specify role for membership

### PR DESCRIPTION
**Pull request by calebthompson**

### CODE REVIEW

File: public/templates/partials/projects/groups_input.html [hunk 1]
```diff
@@ -13,7 +13,14 @@
...
     <li ng-attr-id="{{'group-' + group.id}}" class="group" ng-repeat="group in project.groups">
       {{group.name}}
-      <select ng-model="role"></select>
+      <ui-select ng-model="group.role" theme="selectize" ng-attr-id="{{'group-' + group.id + '-role'}}" on-select="setGroupRole($item, group)">
+        <ui-select-match placeholder="Select a Role">{{$select.selected.name}}</ui-select-match>
+        <ui-select-choices repeat="role in allRoles | filter: $select.search">
+          <strong ng-bind-html="role.name | highlight: $select.search"></strong>
+          <span ng-bind-html="role.description | highlight: $select.search"></span>
+        </ui-select-choices>
+      </ui-select>
```
Comment (calebthompson) [inline public/templates/partials/projects/groups_input.html]: @jntullo this is why I didn't want to use the groups box thing.

Comment (calebthompson) [inline public/templates/partials/projects/groups_input.html]: Video of selecting role, removing group membership to project:

**Comment by chriskacerguis** [comment]
Could you style the group search box a little before the merge?

dashboard_-_jellyfish

**Comment by calebthompson** [comment]
That's actually not something that was touched here: https://jellyfish-core-dev.herokuapp.com/project/4

I believe @sonejah21 has a branch to pretty it up, though.

**Comment by sonejah21** [comment]
I do have a branch that pretties it up :)

**Pull request merged**